github-release.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('@octokit/rest');
+const fs = require('fs');
+
+const accessToken = process.env.GITHUB_TOKEN;
+const appVersion = `v${process.env.npm_package_version}`;
+const buildNamePrefix = `Code.org Maker App-${process.env.npm_package_version}-`;
+const buildNameSuffixes = ['mac.dmg', 'win.exe', 'linux.deb', 'linux.AppImage'];
+
+if (accessToken) {
+  (async() => {
+    const octokit = new Octokit({
+      auth: accessToken,
+    });
+    const release = await createRelease();
+    const releaseId = release.data.id;
+
+    await uploadAssets(releaseId);
+    console.log(`Release drafted. Visit ${release.data.html_url} to publish.`);
+
+    function createRelease() {
+      console.log(`Drafting release ${appVersion}`);
+      return octokit.rest.repos.createRelease({
+        owner: 'code-dot-org',
+        repo: 'browser',
+        tag_name: appVersion,
+        name: appVersion,
+        draft: true,
+      });
+    }
+
+    function uploadAssets(releaseId) {
+      return new Promise(async(resolve, reject) => {
+        for (let i = 0; i < buildNameSuffixes.length; i++) {
+          let buildName = buildNamePrefix + buildNameSuffixes[i];
+          let buildFile = fs.readFileSync(`./dist/${buildName}`);
+          console.log(`Uploading ${buildName}`);
+          await octokit.rest.repos.uploadReleaseAsset({
```

Comment:
should we log an error and/or `reject` if any asset fails to upload?

Reply:
Good catch. Yes, we definitely should. I will add logic to handle failed uploads.

Reply:
@madelynkasula I added error messaging [here](https://github.com/code-dot-org/browser/pull/96/files#diff-2cd718574419de1e5815325176d6e423dedf36fc4ffbe14998aa6d173ec7f6b6R44-R46). I am not rejecting the promise as I think we should continue attempting to upload the remaining assets.